pages/general-information-and-resources/tech-policies/tts-pages.md

```diff
@@ -0,0 +1,84 @@
+
+>Follow this link for more information about using [Cloud.gov Pages](tools/pages)
+
+## Launching a New Static Website at TTS
+
+- [ ] Determine who will serve as the `Website Manager`
+
+>`Website Manager` is defined by the [GSA's Digital.gov Program](https://digital.gov/2023/03/24/who-is-your-website-manager/) 
+
+- [ ] Turn on all [Github Advanced Security](https://docs.github.com/en/get-started/learning-about-github/about-github-advanced-security) for the Site's Repository and `Enable`:
+    - [ ] [CodeQL](https://docs.github.com/en/code-security/code-scanning)
+    - [ ] [Dependabot](https://docs.github.com/en/code-security/dependabot/)
+    - [ ] Secret Scanning
+- [ ] Designate (One) Github Team with `Admin` Access to the repository
+- [ ] Designate (One or More) Github Team(s) with `Write` Access to the repository
+- [ ] Submit a Pull Request to add the Github Repository for the Website to [GSA](https://github.com/GSA/.allstar/blob/main/allstar.yaml) or [GSA-TTS](https://github.com/GSA-TTS/.allstar/blob/main/allstar.yaml) .allstar Github Configuration Scanner)
+- [ ] Create a [`SECURITY.md` file](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository)
+- [ ] Follow GSA's [Digital Lifecycle Program](https://insite.gsa.gov/employee-resources/communications/websites/strategy-policy-and-standards/digital-lifecycle-program?term=DLP) for the Website
+- [ ] Confirm your Website on [Touchpoint's Registry](https://touchpoints.app.cloud.gov/admin/websites) or Submit a new website request in [Touchpoints](https://touchpoints.app.cloud.gov/admin/websites/new)
+
+## Authority to Use (ATU) Review
+Submit an [Authority to Use (ATU) Request](https://github.com/GSA-TTS/tts-pages/issues/new?)
+
+- [ ] Resolve any Critical or High security findings from Security Scanners
+- [ ] Resolve any Web Scan findings provided by GSA Security Team
+
+Once the review is completed the Site Manager will be sent their ATU Approval package via Docusign. The Site Manager will be responsible for managing Security Findings over the lifecycle of the Website.
+
+## Maintaining Approved Sites 
+Sites hosted on TTS Pages are required to have their URLs scanned in accordance with CIO-IT Security-06-30: Managing Enterprise Cybersecurity Risk and GSA’s parameter for National Institute of Standards and Technology (NIST) Special Publication (SP) 800-53, control RA-5, Vulnerability Scanning. 
+
+>This is performed after the ATU request is submitted
+
+## 	Reassessment 
+A Site’s ATU will have to be reassessed and an ATU reissued if the Site is found `NOT` to be in conformity with the requirements within this guide.
```

Comment:
This is passive voice. Who must perform the reassessment? if we start with that, it will be clearer. 

Also, `code formatting` should be swapped for **bold** on the "NOT".

Also, we went from Website to Site here. We should be consistent, especially if we're Capitalizing Things.